docs/DeveloperGuide.md

```diff
@@ -484,25 +489,83 @@ Extensions:
 
 **Use case: UC0X - Create a session**
 
-**Preconditions: Student profile linked to session has been created.**
+MSS:
+
+1. User enters the `add_session` command, together with the session details.
+2. TutorBuddy verifies that the student exists, and the inputs are valid.
```

Comment:
should we be specific and mention session inputs instead?

Reply:
Hmmmm I think cos we are using a general step that is same across the rest of the use cases also, cos you will see this in other use cases also ah. So we probably don't need to specific what type of inputs, since it should be general enough to be use at other places also ah.

Reply:
ok!